src/types/Stats.tsx

```diff
@@ -0,0 +1,5 @@
+export interface Stats {
```

Comment:
Siehe oben zu (a) interface und (b) dem Ort der Dateiablage. IMO ist das hier ein Zeilen, dass Stats einen Ordner braucht in dem Files liegen.